### ISSUE TITLE
java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory not found by org.glassfish.metro.webservices-osgi

### ISSUE THREAD

**Issue by antivalenz** [payara/Payara#1166]
# Description #
----------

Hi!

* I am running an ear-application with a jaxws-based service in a clustered instances setup.
* As soon as the aforementioned service is called, i am getting the stacktrace attached as a plaintext file.
* I am getting this error in the SOAP message handler scope, not the service implementation itself.
* As soon as i copy a jar (e.g. pax-logging-api-1.8.5.jar) containing a recent version of the missing class into the modules directory of an instance and perform a cluster restart, the webservice call is working.
* This seems very related to the issue https://github.com/payara/Payara/issues/493.
* But the issue 493 was marked as fixed by https://github.com/payara/Payara/pull/545.
* If you need any further information, please let me know.

[stacktrace.txt](https://github.com/payara/Payara/files/575513/stacktrace.txt)

## Environment ##

- **Payara Version**: 4.1.1.163
- **Edition**: Full
- **JDK Version**: Oracle JDK 1.8 (`Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)`)
- **Operating System**: `Linux tiny-machine 3.13.0-77-generic #121-Ubuntu SMP Wed Jan 20 10:50:42 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by mulderbaba** [comment]
@antivalenz we reverted #545 since introducing pax-logging revealed more class loading problems than we thought. The applications that were already bundled with SL4J couldn't use our Payara release properly back at the time. So beware that introducing pax-logging could cause the same problems for you. 

The reason behind the NoClassDefFoundError was: metro was using xmlsec classes and bundling them inside its own jar artifact and that specific xmlsec version was demanding LogFactory class on the classpath. That's why I created commons-logging free version of xmlsec which depends on java util logging. It's available at repo: https://github.com/mulderbaba/xmlsec.

The Payara version that you are using 4.1.1.163, depends on webservices-osgi with version 2.3.2-b608.payara-p3. This is the patched version that we already released with Payara 4.1.1.162 and master was merged with PR #672.

I'm sharing my tester, which I deployed against the current master and it worked OK. In order to run the tester can you first create a user with credentials, user: TST, pass: TST? It should be added through the Admin Console under menu: Configurations > server-config > Security > Realms > File and then Manage Users button.
The WSDL URL for the service would be: `http://localhost:8080/ws/EJB1?WSDL`
Sample WS request would be as follows:
```
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ejb="http://ejb.ws.com/">
<soapenv:Header>
<wsse:Security xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd">
<wsse:UsernameToken wsu:Id="UsernameToken-C58CB758D72BB9DB65141779727008327">
<wsse:Username>TST</wsse:Username>
<wsse:Password Type="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-username-token-profile-1.0#PasswordText">TST</wsse:Password>
</wsse:UsernameToken>
</wsse:Security>
</soapenv:Header>
<soapenv:Body>
<ejb:op1>
<arg0>TST</arg0>
</ejb:op1>
</soapenv:Body>
</soapenv:Envelope>
```

If you can also share the simplest reproducer that you could have, I can give that a try.

**Comment by mikecroft** [comment]
Closing since an explanation was given. If you can provide a simple reproducible test case (http://sscce.org/) for us which demonstrates the issue in the latest version of Payara, please reopen this issue
